Add advanced configuration section (#389)
reshufflehq/reshuffle: passport/README.md

  return get(`/todos/${user.id}`);
});
```

### Advanced Configuration

> When using with the reshuffle platform an OAuth provider will be configured. If

Reviewer: ```suggestion
The Reshuffle platform automatically configures an OAuth provider. If
```

> you wish to use an alternative oauth implementation you can override the

Reviewer: ```suggestion
you wish to use an alternative OAuth implementation, override the
```

following environment variables:

- `OAUTH_DOMAIN` \

>   Domain of the OAuth provider (e.g. `oauth.example`)

Reviewer: Not sure this is a good example, it doesn't look like a DNS domain (and, indeed, AFAICT `example.` is not a TLD).

Maybe just give example values for Google or GitHub or some equally popular provider?  Or use `*.example.com`, which _is_ registered by IANA for use in examples.

Author: `example` is a TLD, but it really doesn't look like a TLD...

Reviewer: # Spec fight!  :-) 

https://www.icann.org/resources/pages/tlds-2012-02-25-en doesn't show .example.. Neither does whois.

Author: https://www.iana.org/assignments/special-use-domain-names/special-use-domain-names.xhtml shows `example.` along with `example.com.`, `example.net.` and `example.org`. Yo are right about whois, it doesn't show anything.
